content/updates/2021-03/1-Asset-mass-edit.md

```diff
@@ -0,0 +1,19 @@
+---
+pim_announcement_alt_img: "Asset mass edit screen"
+pim_announcement_audience:
+- EE
+---
+
+# Mass edit your assets
+::: meta-data type="New" features="Productivity" available="late march" in="EE" link-to-doc="../articles/TODO"
+
+If you need to edit several assets from the same family at once, it's now possible to do it in the PIM user interface. No doubt it will save you a tremendous amount of time!
```

Comment:
The two "from" sound weird, doesn't it?

Reply:
```suggestion
If you need to edit attributes of several assets belonging to the same family at once, it's now possible to do it directly from the asset family grid. If some of your attributes have different values per locale, you can also add the attribute several times and select the locale(s) to edit. No doubt it will save you a tremendous amount of time!
```

Reply:
"attributes coming from several assets" sounds weird..